### DISCUSSION TITLE
错误显示

### DISCUSSION THREAD

**Discussion by sjchenhui** [ZuodaoTech/everyone-can-use-english#1011] [General]
重新调整音频和单词的对应后，点击保存会提示错误，需要重新进入刚刚的语言页面才能继续使用
![屏幕截图 2024-08-24 222121](https://github.com/user-attachments/assets/c497ed5a-397f-4877-a7f3-436bd310961c)
![屏幕截图 2024-08-24 222142](https://github.com/user-attachments/assets/a88d2993-9804-4d32-8ee9-686effcbd580)
![屏幕截图 2024-08-24 222148](https://github.com/user-attachments/assets/b581693a-ab31-4857-acb9-a80c3c2b59be)

**Comment by an-lee** [comment]
我检查一下
